src/main/java/com/minecolonies/inventory/GuiHandler.java

```diff
@@ -0,0 +1,33 @@
+package com.minecolonies.inventory;
+
+import com.minecolonies.colony.Field;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.util.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.network.IGuiHandler;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraftforge.fml.relauncher.SideOnly'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)